archivist/assets.py

```diff
@@ -107,14 +109,22 @@ class _AssetsClient:
     def __init__(self, archivist: "type_helper.Archivist"):
         self._archivist = archivist
 
-    def create(self, behaviours: List, attrs: Dict, *, confirm: bool = False) -> Asset:
+    def create(
+        self,
+        behaviours: List,
+        attrs: Dict,
+        *,
+        storage_integrity: StorageIntegrity = StorageIntegrity.TENANT_STORAGE,
+        confirm: bool = False,
+    ) -> Asset:
         """Create asset
 
         Creates asset with defined behaviours and attributes.
 
         Args:
             behaviours (list): list of accepted behaviours for this asset.
             attrs (dict): attributes of created asset.
+            storage_integrity (StorageIntegrity): store asset either on DLT or not
```

Comment:
Instead of 'not' here can we say 'Tenant Storage' and maybe swap 'on' and 'either'?

I'm not sure which is actually preferred, I think I like it being more descriptive though

Reply:
rephrased